components/profile/Profile.js

```diff
@@ -19,9 +19,42 @@ function Profile() {
         <Typography component="h1" variant="h2" gutterBottom>
           Profile
         </Typography>
+        <div>{JSON.stringify(profileData)}</div>
```

Comment:
Happy to remove the work related to initializing the profile data -- I completely expanded the scope of the ticket

Reply:
thanks for the great work! I think it's useful to keep it in this ticket